lib/FusionInventory/Agent/Task/Inventory/Generic/Ipmi/Lan.pm

```diff
@@ -0,0 +1,80 @@
+#
```

Comment:
This file has nothing to do with the original OCSInventory file. You should just get rid of this copyright block.